src/User.php

```diff
@@ -890,6 +895,14 @@ public function post_addItem()
                 $right->add($affectation);
             }
         }
+
+        // Hash user_dn if set
+        if (isset($this->input['user_dn'])) {
```

Comment:
It could be preferable to no hash empty values.
```suggestion
        if (isset($this->input['user_dn']) && is_string($this->input['user_dn']) && strlen($this->input['user_dn']) > 0) {
```